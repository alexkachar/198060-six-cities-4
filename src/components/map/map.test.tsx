import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import mockStore from '../../test-data/mock-store';
import mockOffers from '../../test-data/offers';
import Map from './map';
import Offer from '../../interfaces/offer';

const OFFERS: Offer[] = mockOffers;

it(`Map renders correctly`, () => {
  const tree = renderer
    .create(
        <Provider store={mockStore}>
          <Map
            offers={OFFERS}
            isPropertyMap={false}
          />
        </Provider>, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
