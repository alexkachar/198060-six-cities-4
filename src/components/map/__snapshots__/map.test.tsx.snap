// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map renders correctly 1`] = `
<section
  className="map property__map"
  id="map"
/>
`;
